### PR TITLE
Fixes a typo and some cosmetic changes for migration documentation [ci skip]

### PR DESCRIPTION
**Pull request by abhaynikam**
Related to: https://github.com/rails/rails/pull/40806#discussion_r574388235

cc/ @kamipo @eileencodes

### CODE REVIEW

File: activerecord/lib/active_record/migration.rb [hunk 1]
```diff
@@ -1051,8 +1051,8 @@ def load_migration
   # MigrationContext sets the context in which a migration is run.
   #
   # A migration context requires the path to the migrations is set
-  # in the `migrations_paths` parameter. Optionally a `schema_migration`
-  # class can be provided. For most applications, `SchemaMigration` is
+  # in the +migrations_paths+ parameter. Optionally a +schema_migration+
+  # class can be provided. For most applications, +SchemaMigration+ is
   # sufficient. Multiple database applications need a `SchemaMigration`
```
Comment (jonathanhefner) [inline activerecord/lib/active_record/migration.rb:1056]:
```suggestion
  # sufficient. Multiple database applications need a +SchemaMigration+
```

File: activerecord/lib/active_record/migration.rb [hunk 2]
```diff
@@ -1063,19 +1063,19 @@ def initialize(migrations_paths, schema_migration = SchemaMigration)
...
     # will be run.
     #
-    # If the `current_version` in the schema is less than
-    # the `target_version`, then `down` will be run.
+    # If the +current_version+ in the schema is less than
+    # the +target_version+, then +down+ will be run.
     #
-    # If none of the conditions are met, `up` will be run with
-    # the `target_version`.
+    # If none of the conditions are met, +up= will be run with
```
Comment (jonathanhefner) [inline activerecord/lib/active_record/migration.rb:1077]:
```suggestion
    # If none of the conditions are met, +up+ will be run with
```

**Comment by abhaynikam** [comment]
@jonathanhefner Nice catch. Resolved both comments. Thanks 🎉

**Comment by jonathanhefner** [comment]
Thank you @abhaynikam!

**Pull request merged**